bon/tests/integration/builder_on_fn.rs

```diff
@@ -26,29 +29,27 @@ fn smoke() {
 
         /// Docs for arg2
         arg2: &'_ str,
-        arg3: String,
+        arg3: u64,
```

Comment:
Let's keep the smoke tests under `alloc` feature. They validate into conversions for `String`.

Reply:
Done